docs/tutorials/simple_trading.py.md

```diff
@@ -84,11 +84,11 @@ class SimpleMarketListener(MarketStreamListener):
     futures = [instrument for instrument in instrument_data['data'].values() if instrument['type'] == 'futures'][0]
     selected_futures_id = futures['instrument_id']
 
-  def on_order_book(self, order_book):
+  def on_order_book(self, order_book):  
     if order_book['instrument_id'] != selected_futures_id:
       return 
     bids = order_book['bids']
-    if bids and (not bids[0] or bids[0][0] > sell_threshold):
+    if bids and (not bids[0] or float(bids[0][0]) > sell_threshold):
```

Comment:
I'm not so sure about that. Remember the discussion we had about defensiveness at the client side? If it's in the code we "don't control" (in contrast to the code we "control in runtime" - webapp, where we may have error reporting), the client always relies on what comes out of the API anyway. I'd rather leave defensiveness (or lack thereof) to the decision of the client (at least for now and at this "low level" of API access).

That's even more so when it comes to types - I'd leave it entirely to the client (again - at this level of abstraction) - e.g. if I were writing a trading system, I'd likely use `Decimal` only for accounting-related stuff (account states, open positions) and primitives (floats/integers in satoshis) when it comes to trading data, because pricing algorithms will lose precision anyway and for performance.

We may further discuss it, if needed.